framework/summary_model/src/lib.rs

```diff
@@ -22,28 +21,19 @@ pub struct RunSummaryInitArgs {
     pub wind_tunnel_version: String,
 }
 
-/// Information about the Holochain build used in the run
+/// Build information of any software used in the run.
+/// This is expected to be specific to the binding used by a scenario.
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
-pub struct HolochainBuildInfo {
-    pub cargo_pkg_version: String,
-    pub hdk_version_req: String,
-    pub hdi_version_req: String,
-    pub lair_keystore_version_req: String,
-    pub timestamp: DateTime<Utc>,
-    pub hostname: String,
-    pub host: String,
-    pub target: String,
-    pub rustc_version: String,
-    pub rustflags: String,
-    pub profile: String,
-    pub git_info: Option<HolochainBuildGitInfo>,
-}
+pub struct BuildInfo {
```

Comment:
Maybe that's premature but should this maybe be a HashMap rather so that build info of multiple binaries could be added here?